internal/future/statestore/public.go

```diff
@@ -48,7 +48,7 @@ type Service interface {
 	DeindexTicket(ctx context.Context, id string, indices []string) error
 
 	// FilterTickets returns the Ticket ids for the Tickets meeting the specified filtering criteria.
-	FilterTickets(ctx context.Context, filters []pb.Filter) ([]string, error)
```

Comment:
Once my public_test changes are checked in, please add a test to actually create tickets and filter to see if they return expected results.